### PR TITLE
Ts signals

### PR DESCRIPTION
**Pull request by bjoernma**
Add symbols for DE-ESO:ts1 and ts3 signals and incorporate them into signals.mapcss. Is related to #447 and #449 .

### CODE REVIEW

File: styles/signals.mapcss [hunk 1]
```diff
@@ -747,6 +748,46 @@ node|z17-[railway=signal]["railway:signal:direction"]["railway:signal:minor"="AT
...
+	icon-width: 12;
+	icon-height: 11;
+	text-allow-overlap: true;
+	allow-overlap: true;
+}
+/**************************************************************************/
+/* DE signals for helper engines type Ts                                  */
+/* DE Halt für zurückkehrende Schiebelokomotiven und Sperrfahrten Ts 3    */
+/********************************************/
+node|z14-[railway=signal]["railway:signal:direction"]["railway:signal:helper_engine"="DE-ESO:ts"]["railway:signal:helper_engine:form"=semaphore]["railway:signal:helper_engine:states"="DE-ESO:ts2;DE-ESO:ts3"],
```
Comment (Nakaner) [inline styles/signals.mapcss:774]: Please remove this comma.

File: styles/signals.mapcss [hunk 2]
```diff
@@ -747,6 +748,46 @@ node|z17-[railway=signal]["railway:signal:direction"]["railway:signal:minor"="AT
 	allow-overlap: true;
 }
 
+/********************************************/
+/* DE signals for helper engines type Ts    */
+/* DE Nachschieben einstellen Ts 1          */
+/********************************************/
+node|z14-[railway=signal]["railway:signal:direction"]["railway:signal:helper_engine"="DE-ESO:ts"]["railway:signal:helper_engine:form"=sign]["railway:signal:helper_engine:states"="DE-ESO:ts1"],
```
Comment (Nakaner) [inline styles/signals.mapcss:755]: Please remove the comma.

**Review by Nakaner — CHANGES_REQUESTED** [review]

File: styles/signals.mapcss (hunk 2 above)
Comment (bjoernma) [reply styles/signals.mapcss]: Should be solved. And spotted a missing linebreak / wrong styling for the leading comment.

**Comment by DerDakon** [comment]
Would be nice if you add the icon to the JOSM presets as well. Can be done in it's own PR if this is merged before.

File: josm-presets/de-signals-eso.xml [hunk 3]
```diff
@@ -2181,7 +2181,7 @@ See https://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
 					/>
 			</item>
 			<group name="Signals for Helper Engines (Ts)" de.name="Signale für Schiebelokomotiven und Sperrfahrten (Ts)">
-				<item name="Stop Banking (Ts 1)" de.name="Nachschieben einstellen (Ts 1)" type="node">
+                <item name="Stop Banking (Ts 1)" de.name="Nachschieben einstellen (Ts 1)" icon="de/ts1-32.png" type="node">
```
Comment (DerDakon) [inline josm-presets/de-signals-eso.xml:2184]: Indentation broken.

File: josm-presets/de-signals-eso.xml [hunk 4]
```diff
@@ -2218,7 +2218,7 @@ See https://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
 						de.display_values="Mastbauform,Zwergsignal"
 						/>
 				</item>
-				<item name="Stop/Proceed for Returning Helper Engines (Ts 2/3)" de.name="Halt/Weiterfahrt für zurückkehrende Schieblokomotiven und Sperrfahrten (Ts 2/3)" type="node">
+                <item name="Stop/Proceed for Returning Helper Engines (Ts 2/3)" de.name="Halt/Weiterfahrt für zurückkehrende Schieblokomotiven und Sperrfahrten (Ts 2/3)" icon="de/ts3-32.png" type="node">
```
Comment (DerDakon) [inline josm-presets/de-signals-eso.xml:2221]: Indentation broken.

Comment (bjoernma) [reply josm-presets/de-signals-eso.xml]: Sorry, now my setup is improved so that I should see it beforehand...

**Review by DerDakon — APPROVED** [review]

**Review by Nakaner — APPROVED** [review]

**Comment by DerDakon** [comment]
Please rebase and resolve the commits, and ideally squash some commits together as you would like them to be.

File: styles/signals.mapcss [hunk 5]
```diff
@@ -474,6 +474,7 @@ node|z17-[railway="buffer_stop"]["railway:signal:direction"]["railway:signal:min
 	allow-overlap: true;
 }
 
+
```
Comment (DerDakon) [inline styles/signals.mapcss:477]: extra newline

File: styles/signals.mapcss [hunk 6]
```diff
@@ -759,6 +760,47 @@ node|z17-[railway=signal]["railway:signal:direction"]["railway:signal:minor"="AT
...
+	text-halo-color: white;
+	font-weight: bold;
+	icon-image: "icons/de/ts3-22.png";
+	icon-width: 12;
+	icon-height: 11;
+	text-allow-overlap: true;
+	allow-overlap: true;
+}
+
+
```
Comment (DerDakon) [inline styles/signals.mapcss:803]: extra newline

**Review by DerDakon — CHANGES_REQUESTED** [review]
Almost there ;) We are down to whitespace nitpicks, please squash them into the last commit and then this is ready.

**Comment by bjoernma** [comment]
Thanks for your patience :) Hopefully now it's then better. (and found a third place with a double newline...)

**Review by DerDakon — APPROVED** [review]

**Pull request merged**